src/App.js

```diff
@@ -103,8 +104,12 @@ const App = () => {
   return (
     <div className="App">
       <Routes location={location.state?.backgroundLocation || location}>
-        <Route path="/" element={<IntroChecker />} />
```

Comment:
Please, don't delete intro checker, it's check if it's user first time , if not first time it will redirect user to /dashboards